MVVM Module/Source/Feed Presentation/FeedViewModel.swift

```
@@ -20,11 +20,15 @@ final class FeedViewModel {
 	
 	var onLoadingStateChange: Observer<Bool>?
 	var onFeedLoad: Observer<[FeedImage]>?
+	var onFeedError: Observer<Error>?
```

`onFeedError: Observer<Error>?` could be `onErrorStateChange: Observer<String?>?`, so the ViewModel decides when to show/hide errors and which error string message to show (instead of the view controller). It's easier to manage presentation state transitions when all changes are centralized.

For example, in the `loadFeed()` method, the ViewModel can call  `onErrorStateChange(.none)` to hide the error before loading the feed.

And when there's an error, the ViewModel should call the closure with the formatted error string `onErrorStateChange(Localized.Feed.loadError)`

---

fixed